.gitignore

```diff
@@ -6,12 +6,16 @@
 
 .idea/misc.xml
 
+.idea/preferred-vcs.xml
```

Comment:
When I switched to working from the Mac to my Windows laptop, that file appeared. The Windows version of Webstorm created that file. In theory it should be harmless, and [JetBrains docs](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) don't say anything about it, but these are its contents:
````xml
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="PreferredVcsStorage">
    <preferredVcsName>ApexVCS</preferredVcsName>
  </component>
</project>
````
It seems to relate to a [source control system](https://community.oracle.com/thread/4038126) of an [Oracle development environment](https://apex.oracle.com/en/) that I don't have installed. I don't remember seeing that file when we reviewed the contents of the .idea/ folder back when I was in Mac OS, so I chose to add it to .gitignore.